selfdrive/ui/translations/main_pt.ts

```diff
@@ -0,0 +1,1307 @@
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="200"/>
+        <source>right</source>
+        <translation>direita</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="211"/>
+        <source>Are you sure you want to reboot?</source>
+        <translation>Tem certeza que quer reiniciar?</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="218"/>
+        <source>Disengage to Reboot</source>
+        <translation>Desacione para Reiniciar</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="224"/>
+        <source>Are you sure you want to power off?</source>
+        <translation>Tem certeza que quer desligar?</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="231"/>
+        <source>Disengage to Power Off</source>
+        <translation>Desacione para Desligar</translation>
+    </message>
+</context>
+<context>
+    <name>DriveStats</name>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="37"/>
+        <source>Drives</source>
+        <translation>Dirigidas</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>Horas</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>TOTAL</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>SEMANA PASSADA</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>KM</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>Milhas</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="55"/>
+        <source>camera starting</source>
+        <translation>camera iniciando</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>Cancelar</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation>
+            <numerusform>Necessita no mínimo %n caractere(s)!</numerusform>
+            <numerusform></numerusform>
```

Comment:
I just noticed this, can we make sure all the numerus translations have plural forms?

Reply:
Now I think I understand what was missing. I had thought i was right the first time but had not seen that a new field to be filled in the linguist had emerged.